Katas/src/at/codersbay/katas/Main.java

```diff
@@ -1,15 +1,31 @@
 package at.codersbay.katas;
 
+
+
 public class Main {
 
-    public static void main(String[] args) {
+
+
+	
+
+
+	public static void main(String[] args) {
 
         System.out.println("After you finished your Kata, make sure to call it from the main class to show us what you've done!");
 
         // falls du eine TicTacToe klasse geschrieben hast die mit einer statischen Methode ein Tic Tac Toe feld auf die Konsole drucken kann könnte das so aussehen:
         // eg.: TicTacToe.printRandomField();
         // eg.: TicTacToe.printField("X", "", "O", "", "", "", "X", "X", "O");
 
+        ArrayMean arrayMean = new ArrayMean();
+        ArrayMean.calcArravg();
```

Comment:
according to JavaCodeConvention please rename your method to Array.Mean.calcArrAvg()